src/rrdp_iostat.ml

```diff
@@ -480,19 +480,35 @@ module Stats_value = struct
 		| Some s3 ->
 				let last_s3 = match last_stats_blktap3 with
```

Comment:
you can remove this `let last_s3 = match last_stats_blktap3 ...` since you've already got the fold

Reply:
I don't think so. The `fold` takes the option value (using `last_stats_blktap3`) but there are other places in the code where it needs the unwrapped value. This is one of the differences to the original code.

Reply:
okay, nevermind then!